### ISSUE TITLE
Комментарии к пользовательским сценариям

### ISSUE THREAD

**Issue by aakuptsov** [THunderEmperORMike22/IT-Application-repo#4]
>>	В зависимости от технического задания – предоставление возможности экспорта ответов пользователя в файл указанного заказчиком формата (pdf, doc) c сохранением в облаке

**Необходимо добавить этот сценарий** в список пользовательских сценариев.

Если это бот для внутрикорпоративной коммуникации, то необходимо сделать группирование пользователей по департаментам / отделам с возможностью создания объявлений в департаменте / отделе, а также коммуникацией между департаментами. 

Ещё необходимо добавить возможность отправки новостей из HR другим сотрудникам на определённые темы, например, о том, что сотрудник направлен на тренинг и т. п.

**Comment by EmperorMikhail223** [comment]
Здравствуйте! 
По поводу внутрикорпоративной коммуникации, я обсуждал это со службой безопасности, но пока не пришел к договоренности. Это государственная корпорация и у них все строго с персональными данными сотрудников, так как я официально не включён в штат. Но, я постараюсь договориться.

**Comment by aakuptsov** [comment]
Я понимаю, но тут есть нюанс: поскольку всё-таки это проект выносится на защиту как ИВР, мы тоже имеем право голоса. Как вариант, ты можешь сделать 2 версии бота: одна пойдёт заказчику, а другая, в которую ты и включишь внутрикорпоративную коммуникацию - на защиту.

**Comment by EmperorMikhail223** [comment]
Хорошо, а людей можно придумать?

**Comment by aakuptsov** [comment]
Да, конечно. Тестовые аккаунты. Плюс мы их и создадим прямо на защите :) 

Кстати, тогда **добавь, пожалуйста, сценарий удаления сотрудника**.

**Comment by EmperorMikhail223** [comment]
Хорошо.
